hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/NettyGrpcServerManager.java

```diff
@@ -114,9 +115,14 @@ public NettyGrpcServerManager(
         requireNonNull(queryWorkflow);
         requireNonNull(metrics);
 
-        // Convert the various RPC service definitions into transaction or query endpoints using the GrpcServiceBuilder.
+        // Convert the various RPC service definitions into transaction or query endpoints using the
+        // GrpcServiceBuilder.
         services = servicesRegistry.registrations().stream()
                 .map(ServicesRegistry.Registration::service)
+                // Not all services are RPC services, but here we need RPC services only. The main difference
+                // between RPC service and a service is that the RPC service has RPC definition.
+                .filter(v -> v instanceof RpcService)
```

Comment:
Could you please explain how it's better?

Reply:
>if v were of type (GrpcService extends RpcService), instanceof would fail. 

I don't think it is correct. `instanceof` operator (unlike `getClass()` equality) checks if the object's class is exactly the same as the provided class **or** the object's class has the provided class as parent. In this particular case the requirement for the object to implement `RpcService`. 

`isAssignableFrom(...)` expression may be beneficial, if we don't know the class we want against at compile time. Here it's not the case.

Reply:
> or the object's class has the provided class as a parent
Huh, maybe this is a change in one of the more recent Java versions. 

I still think `isAssignableFrom(...)` is also correct, but there's no need to go on about it here :) thanks for checking!